log/zap.go

```diff
@@ -46,12 +49,30 @@ func NewLoggerZap(config ZapConfig) (*Zap, error) {
 		return nil, errors.Wrap(err, "error on building zap logger")
 	}
 
+	if appEnvironment() == "local" {
+		loggerConfig.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+
+		logger = zap.New(
+			zapcore.NewCore(
+				zapcore.NewConsoleEncoder(loggerConfig.EncoderConfig),
+				zapcore.AddSync(colorable.NewColorableStdout()),
+				loggerConfig.Level,
+			),
+		)
+	}
+
 	return &Zap{
 		logger: logger,
 		tracer: config.Tracer,
 	}, nil
 }
 
+func appEnvironment() string {
+	env := os.Getenv("ENV")
+
+	return strings.ToLower(env)
+}
+
```

Comment:
Pq não usa a variável DEBUG? Sempre que rodamos local o ideal o DEBUG=true
Outra duvida, se deixar ligado sempre como fica o output no grafana? Vai surgir aqueles caracteres especiais de terminal (é por isso que vc só deixou ligado na maquina do dev)? Se for essa a justificava vale colocar um comentário justificando pq não é claro a justificativa.